conda_lockfiles/dumpers/__init__.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import annotations
+
+from .conda_lock_v1 import export_to_conda_lock_v1
+
+__all__ = ["export_to_conda_lock_v1", "LOCKFILE_FORMATS"]
+
+LOCKFILE_FORMATS: dict[str, callable] = {
+    "conda-lock-v1": export_to_conda_lock_v1,
+}
```

Comment:
Should we mirror the loaders OOP design with the `supports` method?

Reply:
I don't thinking the `supports` pattern is a good option for exporting. 

For loaders, the input is a file which is in an unknown format. Each loader examines the input file and determines if it is in the format they support. Each input will have a single (or no) loader which supports it.

For exporting the input is a prefix to a conda environment. Every dumper is capable of creating a lockfile from the input so a `supports` method would always return True. The input format could be used to establish support. With this approach the `supports` method would be a comparison which seems like an unnecessary complication.